bs/src/tablecloth.mli

```diff
@@ -7,71 +7,49 @@ Function names that are in snake_case have their documentation written in OCaml
 Function names that are in camelCase have their documentation written in ReasonML format.
 *)
 
+val ( <| ) : ('a -> 'b) -> 'a -> 'b
 (**
-  The `<|` operator applies a function to an argument. It is equivalent to the `@@` operator,
+  The [<|] operator applies a function to an argument. It is equivalent to the [@@] operator,
   and its main use is to avoid needing extra parentheses.
 
-  ### Example
-
-  ```ocaml
+  {[
   let sqr x = x * x
   let result = sqr <| 25 (* 625 *)
-  ```
-
-  ```reason
-  let sqr = (x) => {x * x};
-  let result = sqr <| 25  /* 625 */
-  ```
+  ]}
 *)
-val ( <| ) : ('a -> 'b) -> 'a -> 'b
 
+val ( >> ) : ('a -> 'b) -> ('b -> 'c) -> 'a -> 'c
 (**
-    The `>>` operator returns a function that is the equivalent of the composition of its function arguments.
-    The main use of `>>` is to avoid writing parentheses.
-
-  `(f >> g) x` (`(f >> g)(x)` in ReasonML) is the equivalent of `f (g x)` (`f(g(x))` in ReasonML)
+    The [>>] operator returns a function that is the equivalent of the composition of its function arguments.
+    The main use of [>>] is to avoid writing parentheses.
 
-  ### Example
+  [(f >> g) x] is the equivalent of [f (g x)]
 
-  ```ocaml
+  {[
 
   let f = (sqrt >> floor);;
   (f 17.0) = 4.0
-  ```
-
-  ```reason
-  let f = sqrt >> floor;
-  f (17.0) == 4.0;
-  ```
+  ]}
 *)
-val ( >> ) : ('a -> 'b) -> ('b -> 'c) -> 'a -> 'c
 
+val ( << ) : ('b -> 'c) -> ('a -> 'b) -> 'a -> 'c
 (**
-  The `<<` operator returns a function that is the equivalent of the reverse composition of its function arguments.
-
-  `(f << g) x` (`(f << g)(x)` in ReasonML) is the equivalent of `g (f x)` (`g(f(x))` in ReasonML)
+  The [<<] operator returns a function that is the equivalent of the reverse composition of its function arguments.
 
-  ### Example
+  [(f << g) x] is the equivalent of [g (f x)]
```

Comment:
...and this the other way: `[(f << g) x] is the equivalent of [f (g x)]`
The docs interchanged signatures for forward and backward composition